security/providers/common/src/main/java/io/helidon/security/providers/common/OutboundTarget.java

```diff
@@ -101,7 +101,13 @@ private OutboundTarget(Builder builder) {
         }
     }
 
-    static OutboundTarget create(Config c) {
+    /**
+     * Create a target from configuration.
+     *
+     * @param c configuration on the node of a single outbound target
```

Comment:
@param c configuration (is that 'c' correct here?)

Reply:
Renamed to `config`, as `c` does not look good here

Reply:
Nevertheless for Javadoc it must be the same as is the parameter name of the method